src/TextBox.js

```diff
@@ -33,6 +33,7 @@ export default class TextBox extends BaseClass {
 
     super();
 
+    this._ariaHidden = "false";
```

Comment:
this needs to use the constant function: `this._ariaHidden = constant("false");`